src/components/pages/TabPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { ScrollView, StyleSheet } from 'react-native';
+import { Header, Icon, normalize } from 'react-native-elements';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const styles = StyleSheet.create({
+  header: {
+    backgroundColor: '#00bfff',
+  },
+  page: {
+    flexGrow: 1,
+    backgroundColor: '#00bfff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
+
+const Page: React.FC<Props> = ({ children }) => {
```

Comment:
これで大丈夫だと思います（`Props`は定義しなくてもいいかと）
```suggestion
const Page: React.FC<PropsWithChildren<object>> = ({ children }) => {
```

Reply:
ありがとうございます！勉強になりました！変更します！

Reply:
変更しました！